docs/tutorial/tutorial.md

```diff
@@ -206,7 +206,7 @@ The React Devtools extension for [Chrome](https://chrome.google.com/webstore/det
 
 It lets you inspect the props and state of any of the components in your tree.
 
-It doesn't work great on CodePen because of the multiple frames, but if you log in to CodePen and confirm your email (for spam prevention), you can go to Change View > Debug to open your code in a new tab, then the devtools will work. It's fine if you don't want to do this now, but it's good to know that it exists.
+It doesn't work great on CodePen because of the multiple frames, but you can try it on <a href="/react/tutorial/debug-view.html" target="_blank">on the current code</a> or <a href="/react/tutorial/debug-view-final.html" target="_blank">the final result</a>.
```

Comment:
It works great when forking though so maybe we should encourage people to fork. In my experience people trying React often already have CodePen accounts.